go/chat/mobilepush.go

```diff
@@ -75,79 +73,26 @@ func (h *MobilePush) AckNotificationSuccess(ctx context.Context, pushIDs []strin
 	}
 }
 
-func (h *MobilePush) formatTextPush(ctx context.Context, uid gregor1.UID, convID chat1.ConversationID,
-	membersType chat1.ConversationMembersType, msg chat1.MessageUnboxed) (res string, err error) {
-	switch membersType {
-	case chat1.ConversationMembersType_TEAM:
-		var channelName string
-		// Try to get the channel name
-		ib, _, err := h.G().InboxSource.Read(ctx, uid, types.ConversationLocalizerBlocking,
-			types.InboxSourceDataSourceAll, nil,
-			&chat1.GetInboxLocalQuery{
-				ConvIDs: []chat1.ConversationID{convID},
-			}, nil)
-		if err != nil || len(ib.Convs) == 0 {
-			h.Debug(ctx, "FormatPushText: failed to unbox conv: %v", convID)
-		} else {
-			channelName = ib.Convs[0].Info.TopicName
-		}
-		if channelName == "" {
-			// Don't give up here, just display the team name only
-			h.Debug(ctx, "FormatPushText: failed to get topicName")
-			return fmt.Sprintf("%s (%s): %s", msg.Valid().SenderUsername,
-				msg.Valid().ClientHeader.TlfName, msg.Valid().MessageBody.Text().Body), nil
-		}
-		return fmt.Sprintf("%s (%s#%s): %s", msg.Valid().SenderUsername,
-			msg.Valid().ClientHeader.TlfName, channelName,
-			msg.Valid().MessageBody.Text().Body), nil
-	default:
-		return fmt.Sprintf("%s: %s", msg.Valid().SenderUsername, msg.Valid().MessageBody.Text().Body), nil
-	}
-}
+func (h *MobilePush) GetConvInfo(ctx context.Context, uid gregor1.UID, convID chat1.ConversationID) (info chat1.ConversationInfoLocal, err error) {
```

Comment:
Can delete this function, just use `utils.GetVerifiedConv` instead and call it from `bind`